src/ducks/loan/LoanDetailsPage.jsx

```diff
@@ -114,69 +106,79 @@ const KeyInfosSection = translate()(props => {
       <Row
         type="amount"
         title={t('LoanDetails.keyInfos.borrowedAmount')}
-        value={get(account, 'loan.usedAmount')}
+        value={borrowedAmount}
       />
       <Row
         type="amount"
         title={t('LoanDetails.keyInfos.remainingCapital')}
-        value={-get(account, 'balance')}
+        value={Math.abs(balance)}
       />
       <Row
         type="rate"
         title={t('LoanDetails.keyInfos.interestRate')}
-        value={get(account, 'loan.rate')}
+        value={rate}
       />
     </Section>
   )
 })
 
-const PaymentsSection = translate()(props => {
+export const PaymentsSection = translate()(props => {
   const { account, t, f } = props
 
-  const shouldRender = hasSomeInfos(account, [
-    'loan.lastPaymentDate',
-    'loan.nextPaymentDate'
-  ])
+  const lastPaymentAmount = get(account, 'loan.lastPaymentAmount')
+  const nextPaymentAmount = get(account, 'loan.nextPaymentAmount')
+
+  const shouldRender = [lastPaymentAmount, nextPaymentAmount].some(
+    isExistingData
+  )
 
   if (!shouldRender) {
     return null
   }
 
+  const lastPaymentDate = get(account, 'loan.lastPaymentDate')
+  const nextPaymentDate = get(account, 'loan.nextPaymentDate')
+
   return (
     <Section title={t('LoanDetails.payments.title')}>
       <Row
         type="amount"
         title={t('LoanDetails.payments.lastPayment')}
         value={get(account, 'loan.lastPaymentAmount')}
         caption={
-          t('LoanDetails.dateGlue') +
-          ' ' +
-          f(get(account, 'loan.lastPaymentDate'), DATE_FORMAT)
+          lastPaymentDate
+            ? `${t('LoanDetails.dateGlue')} ${f(lastPaymentDate, DATE_FORMAT)}`
+            : undefined
         }
       />
       <Row
         type="amount"
         title={t('LoanDetails.payments.nextPayment')}
         value={get(account, 'loan.nextPaymentAmount')}
         caption={
-          t('LoanDetails.dateGlue') +
-          ' ' +
-          f(get(account, 'loan.nextPaymentDate'), DATE_FORMAT)
+          nextPaymentDate
+            ? `${t('LoanDetails.dateGlue')} ${f(nextPaymentDate, DATE_FORMAT)}`
+            : undefined
         }
       />
     </Section>
   )
 })
 
-const CharacteristicsSection = translate()(props => {
+export const CharacteristicsSection = translate()(props => {
   const { account, t } = props
 
-  const shouldRender = hasSomeInfos(account, [
-    'loan.subscriptionDate',
-    'loan.maturityDate',
-    'loan.nbPaymentsLeft',
-    'loan.nbPaymentsDone'
-  ])
+  const subscriptionDate = get(account, 'loan.subscriptionDate')
+  const maturityDate = get(account, 'loan.maturityDate')
+  const nbPaymentsLeft = get(account, 'loan.nbPaymentsLeft')
+  const nbPaymentsDone = get(account, 'loan.nbPaymentsDone')
+
+  const shouldRender = [
+    subscriptionDate,
+    maturityDate,
+    nbPaymentsLeft,
+    nbPaymentsDone
+  ].some(isExistingData)
 
```

Comment:
Just checking, is some correctly supported by the browsers we target ?

Reply:
Yes